### PR TITLE
fix default leader election type

### PR DESCRIPTION
**Pull request by mlmhl**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

The valid leader election types are `configmaps` and `leases`, but the default value of flag `leaderElectionType` is `configmap`, which is illegal. This PR changes the default value to `configmaps`.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
The default leader election type will be `configmaps` if not specified in the command line
```

### CODE REVIEW

File: cmd/csi-attacher/main.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ var (
 	retryIntervalMax   = flag.Duration("retry-interval-max", 5*time.Minute, "Maximum retry interval of failed create volume or deletion.")
 
 	enableLeaderElection    = flag.Bool("leader-election", false, "Enable leader election.")
-	leaderElectionType      = flag.String("leader-election-type", "configmap", "the type of leader election, options are 'configmaps' (default) or 'leases' (recommended). The 'configmaps' option is deprecated in favor of 'leases'.")
+	leaderElectionType      = flag.String("leader-election-type", "configmaps", "the type of leader election, options are 'configmaps' (default) or 'leases' (recommended). The 'configmaps' option is deprecated in favor of 'leases'.")
```
Comment (msau42) [inline cmd/csi-attacher/main.go]: Can we define constants for these? It could help avoid typos in the future

Comment (mlmhl) [reply cmd/csi-attacher/main.go]: Good suggestion, already updated, PTAL :)

**Pull request merged**